dotCMS/src/main/java/com/dotcms/rest/api/v1/container/ContainerResource.java

```diff
@@ -673,16 +675,26 @@ public final Response saveNew(@Context final HttpServletRequest  request,
         container.setShowOnMenu(containerForm.isShowOnMenu());
         container.setTitle(containerForm.getTitle());
 
-        if(containerForm.getMaxContentlets() == 0){
+        if (containerForm.getMaxContentlets() == 0) {
             container.setCode(containerForm.getCode());
         }
 
-        this.containerAPI.save(container, containerForm.getContainerStructures(), host, user, pageMode.respectAnonPerms);
+        this.containerAPI.save(container, containerForm.getContainerStructures(), host, user,
```

Comment:
you are calling save + publish
both must be encapsulated on one transaction to avoid weird state cases on the db

Reply:
it has been addressed